tiledb/ml/readers/pytorch.py

```diff
@@ -33,22 +43,30 @@
 
 def PyTorchTileDBDataLoader(
     *array_params: ArrayParams,
-    batch_size: int,
+    dataloader_params: Dict[str, Any] = {},
```

Comment:
Any reason for passing the extra params as a dict instead of kwargs (``**dataloader_params``)? I'm slightly more keen on the latter but no strong opinion.

Reply:
Nope. Will change it.